crates/biome_js_analyze/tests/specs/nursery/useExplicitType/invalidDeclarationStatements.ts.snap

```diff
@@ -15,37 +14,3 @@ export {
 }
 
 ```
-
-# Diagnostics
-```
-invalidDeclarationStatements.ts:1:14 lint/nursery/useExplicitType ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
-
-  × The variable doesn't have a type defined.
-  
-  > 1 │ export const foo = {};
-      │              ^^^
-    2 │ 
-  
-  i Declaring the type makes the code self-documented and can speed up TypeScript type checking.
-  
-  i Add a type to the variable.
```

Comment:
Same thing:
- if it's a regression, we should fix it
- if it's not a regression, we should remove it from the source code